app/sidekiq/benefits_intake_status_job.rb

```diff
@@ -12,21 +12,33 @@ def perform
                                   .where(form_submission_attempts: { aasm_state: 'pending' })
                                   .map(&:benefits_intake_uuid)
     response = BenefitsIntakeService::Service.new.get_bulk_status_of_uploads(pending_form_submission_ids)
-    handle_response(response)
-    submissions_handled = response.body['data'].count
-    Rails.logger.info({ name: 'BenefitsIntakeStatusJob ended', submissions_handled: })
+    stats = handle_response(response)
+    Rails.logger.info({ message: 'BenefitsIntakeStatusJob ended' }.merge(stats))
   end
 
   private
 
   def handle_response(response)
+    stats = {
+      total_submissions_handled: 0,
+      pending_submissions_handled: 0,
+      failed_submissions_handled: 0,
+      successful_submissions_handled: 0
+    }
     response.body['data'].each do |submission|
       if submission.dig('attributes', 'status') == 'error' || submission.dig('attributes', 'status') == 'expired'
+        stats[:failed_submissions_handled] += 1
         handle_failure(submission)
       elsif submission.dig('attributes', 'status') == 'vbms'
+        stats[:successful_submissions_handled] += 1
         handle_success(submission)
+      else
+        stats[:pending_submissions_handled] += 1
       end
+      stats[:total_submissions_handled] += 1
```

Comment:
How many total submissions do you expect? I'm wondering if efficiency should be considered.

Reply:
We don't have a firm grip on that yet, but we plan to monitor system as we onboard each new form.
We are also working on requirements to auto-fail any submission that doesnt move to fail/success after a set amount of time.


Possible solutions: working in batches or separate jobs per form.

Reply:
Currently in the low hundreds so likely not a concern yet, as Thomas points out, but we'll be monitoring it.